### ISSUE TITLE
Enable developers to connect their Active Azure Directory to Auth

### ISSUE THREAD

**Issue by amchew** [microsoft/appcenter#994]
We continue to expand Auth's identity management provider offerings. Last month we started work on [enabling developers to connect their Firebase and Auth0 tenants](https://github.com/microsoft/appcenter/issues/637), and this month we'll be focusing on the work to enable developers to connect their Azure Active Directory (AAD) tenants. We've heard the feedback from our enterprise users that connecting their AAD tenants to Auth is key to using Auth, Data and Push, and we're excited to ship this feature soon! 

**We're actively looking for users for beta testing**, feel free to leave a comment below if you have any feedback or are interested to share feedback!

Here are some sample designs:
![image](https://user-images.githubusercontent.com/33160470/64455420-7b189a00-d0a2-11e9-9d26-9d46185dc69b.png)
![image](https://user-images.githubusercontent.com/33160470/64455465-91bef100-d0a2-11e9-8c7d-9dc4180fd8f8.png)
![image](https://user-images.githubusercontent.com/33160470/64455482-a13e3a00-d0a2-11e9-8912-37cbc808a5a9.png)
![image](https://user-images.githubusercontent.com/33160470/64455536-c337bc80-d0a2-11e9-8fe5-abaa7db4d2fd.png)
![image](https://user-images.githubusercontent.com/33160470/64455563-d8145000-d0a2-11e9-9b6b-5cbe15df2ad8.png)

**Comment by askpatrickw** [comment]
I've been testing Auth and found a few issues. The high-level setup is a single GitHub repository with a ReactNative app started from react-native  CLI. I've hookedup  two AppCenter Apps. Oner for Android and one for iOS.

**Android**
Android was connected to my AD B2C directory and my app redirects to the Web Based  AD B2C login.  When I go to the settings for my Auth connection, the pane is blank.

![image](https://user-images.githubusercontent.com/4002194/71118995-ff90b380-218d-11ea-9e71-03af99e967d2.png)


**iOS** 
I cannot connect my iOS app to AD B2C, it cannot see my directory. After selecting the  subscription is reports no tenants found. I assume this is an order of operations problem and had I setup iOS first then Android would not have worked.

![image](https://user-images.githubusercontent.com/4002194/71119144-5d250000-218e-11ea-8568-ae622e35723d.png)


**"Native" React Native Screens**
There also needs to be support for allowing authentication from the App itself, its own native screens, in addition to the redirect to web browser. This is the MAIN use case for app developers. Redirecting  to a browser is bad brand experience for my clients. I'd be happy with a working sample a RN library I can add to my project which keys off the configured VS  App Center settings.

**Comment by Zakeelm** [comment]
Thank you so much for all of your feedback. We announced this week in a [blog post]( https://devblogs.microsoft.com/appcenter/app-center-mbaas-retirement/) that we’re retiring the Auth and Data services later this year and recommending customers transition to Azure Active Directory B2C and Azure Cosmos DB instead. Let me know if you have any additional questions about this transition and i'll make sure to get them answered! Closing this issue.
